### ISSUE TITLE
[BUG] - typo in alt of HomeSection.jsx

### ISSUE THREAD

**Issue by xscottxbrownx** [codeforpdx/PASS#444]
### Bug Description:
The images used in `HomeSection.jsx` should have alt tags for accessibility and seo, but the are misspelled to `lt` so are not working as intended.

---
### Expected Behavior:
Replace typo with `alt` so it will work as intended.
